cleanlab_studio/errors.py

```diff
@@ -24,3 +24,7 @@ def __init__(self) -> None:
 class AuthError(APIError):
     def __init__(self) -> None:
         super().__init__("invalid API key")
+
+
+class CleansetRunError(RuntimeError):
```

Comment:
Do we want a special exception for this case, for this single function? Maybe a more general exception that indicates that something went wrong, and that it's a permanent failure. Could be helpful, e.g., for a `deploy_model()` method where something goes wrong during training.

Could do, e.g., `class InternalError(Exception)`

Reply:
I expect we might want to use this error in the future in other places -- for example, a `train` method that takes in a pandas dataframe, uploads it, creates project, waits for and returns results.

I'll create an `InternalError` base class though and have this inherit from it

Reply:
I expect we might want to use this error in the future in other places -- for example, a `train` method that takes in a pandas dataframe, uploads it, creates project, waits for and returns results.

I'll create an `InternalError` base class though and have this inherit from it

Reply:
Ok. For this one, I'd suggest renaming to `CleansetError`